core/bin/zksync_api/src/fee_ticker/validator/types.rs

```diff
@@ -0,0 +1,26 @@
+use serde::{Deserialize, Serialize};
+
+// --------- locally simplified Contract struct for retreiving market data only
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
```

Comment:
Im not sure about this, but we may discuss it or I may be confused, but do we need the Clone, derive?

Reply:
1. this is copied from CoinGecko rust sdk (I know that doesn't mean we should keep it)
2. It would be safer, but also more annoying to work with. 

Say we want to pass it to two functions. Ideally, w'd just be passing the reference to it. But if we have to pass it to a function that takes it by value, then we have to clone, or it will be moved to that function. If it isn't returned by this function, then it would be lost when the function ends.

Reply:
Thanks for the explanation :)